src/SignalR/common/Shared/PipeWriterStream.cs

```diff
@@ -1,12 +1,9 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Buffers;
-using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Internal;
```

Comment:
Revert changes here unless a namespace is disappearing for some reason. Same for other `using ...Internal` cleanup.

And, if namespaces are disappearing, why?

Reply:
These aren't used. The Microsoft.AspNetCore.Internal is no longer available since the internal APIs are not generated by default so this using must be removed. This originally was added to support an internal extension which has since been removed.

Reply:
> the internal APIs are not generated by default so this using must be removed

Is this a change in the build? Were warnings and errors suppressed somehow before your changes?

Reply:
So the history of this change is:
1. ValueTaskExtensions was added in https://github.com/aspnet/AspNetCore/commit/9de42d516e6ee9c8eb17700e134f3ac8a7a682ff to the .Internal namespace
2. This was removed later when the extension method was no longer needed but the using statement was not removed.
3. The Microsoft.AspNetCore.Internal namespace still existed since other internal APIs from unrelated transitive references still contained this namespace. 
4. Changing to ref assemblies meant that this namespace no longer exists for most cases. In this case, no internal APIs were being used so I'm removing the using statements.